script.module.t1mlib/lib/t1mlib.py

```diff
@@ -126,9 +128,53 @@ def getAddonLiveVideo(self, url):
     def getAddonVideo(self, url):
         xbmcplugin.setResolvedUrl(int(sys.argv[1]), True, xbmcgui.ListItem(path=uqp(url), offscreen=True))
 
+
     def doFunction(self, url):
         return
 
+
+    def cleanFilename(self, filename):
```

Comment:
why not xbmcvfs.makeLegalfilename?

Reply:
After extensive testing, it's better for cross platform .strm file use to use this routine.